### PR TITLE
Update data table to Polaris v3.10.0

### PR DESCRIPTION
**Pull request by andrewpye**
Adds the new `data-polaris-header-cell` attribute to `polaris-data-table` cells with `header=true`.

### CODE REVIEW

**Comment by andrewpye** [comment]
[Upgrade ember-polaris to v3.10.0](https://trello.com/c/orVDHTEo/320-upgrade-ember-polaris-to-v3100)

**Comment by vladucu** [comment]
the diff gives me a bunch of changes....are those just shifting things around? nothing really changed in functionality?

**Comment by andrewpye** [comment]
>the diff gives me a bunch of changes....are those just shifting things around? nothing really changed in functionality?

From what I could tell, the vast majority of the diff comes from a) moving things around b) adding i18n stuff, which works out with the same result as what we've already got here.

**Comment by andrewpye** [comment]
@vladucu but if you think there's something I've missed, let me know! 🙏

**Comment by andrewpye** [comment]
@vladucu yep got the right one. Made a couple of tweaks, wondering if there's anything else you think I've missed?

**Comment by andrewpye** [comment]
@vladucu _bump_ 😅

**Comment by vladucu** [comment]
ops, sorry
on it in 10 mins

File: addon/utils/data-table.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export function measureColumn(tableData) {
```
Comment (vladucu) [inline addon/utils/data-table.js]: Some slight difference to the original here
Screen Shot 2019-04-04 at 9 01 27 PM

**Comment by andrewpye** [comment]
> Is there anything we need to update related to the changes below
> Screen Shot 2019-04-04 at 8 56 20 PM
> ?

The `sortDirection = defaultSortDirection` change doesn't seem to be important since they reassign `sortDirection` before it gets used 🤷‍♂️Adding it anyway though in case a future change in this area needs it and we miss it.

`sorted` doesn't seem to be used at all, so I'm just going to remove it #_yolo_ 😅

**Comment by vladucu** [comment]
> The sortDirection = defaultSortDirection change doesn't seem to be important since they reassign sortDirection before it gets used 🤷‍♂️Adding it anyway though in case a future change in this area needs it and we miss it.
sorted doesn't seem to be used at all, so I'm just going to remove it #yolo 😅

figured it's something along those lines 👍

**Pull request merged**